katas/content/qubit/index.md

```diff
@@ -165,6 +192,139 @@ This demo shows how to allocate a qubit and examine its state in Q#. This demo u
+    "descriptionPath": "./learn_single_qubit_state/index.md",
+    "codePaths": [
+        "../KatasLibrary.qs",
+        "./learn_single_qubit_state/Verification.qs"
+    ],
+    "placeholderSourcePath": "./learn_single_qubit_state/Placeholder.qs",
+    "solutionPath": "./learn_single_qubit_state/solution.md"
+})
+
+@[section]({
+    "id": "qubit_visualizing_multi_qubit",
+    "title": "Display the Quantum State of a Multi-Qubit Program"
+})
+
+Now let's take a look at the general case: a program that acts on $N$ qubits. 
+The state of the quantum system used by this program can be represented as a complex vector of length $2^N$, or, using Dirac notation,
+
+$$\begin{bmatrix} x_0 \\ x_1 \\ \vdots \\ x_{2^N-1}\end{bmatrix} = \sum_{k = 0}^{2^N-1} x_k |k\rangle$$
+
+Same as in the single-qubit case, `DumpMachine` allows you to see the amplitudes $x_k$ for all basis states $|k\rangle$ directly.
+
+> Note the use of an integer in the ket notation instead of a bit string with one bit per qubit. 
+By default, `DumpMachine` uses little-endian to convert bit strings to integers in the ket notation.
+For more details on endiannes, see [the multi-qubit systems tutorial](../MultiQubitSystems/MultiQubitSystems.ipynb#Endianness).
+
+## Demo: DumpMachine for multi-qubit systems
+
+@[example]({"id": "multi_qubit_dump_machine_demo", "codePath": "./examples/MultiQubitDumpMachineDemo.qs"})
+
+@[exercise]({
+    "id": "learn_basis_state_amplitudes",
+    "title": "Learn Basis State Amplitudes",
+    "descriptionPath": "./learn_basis_state_amplitudes/index.md",
+    "codePaths": [
+        "../KatasLibrary.qs",
+        "./learn_basis_state_amplitudes/Verification.qs"
+    ],
+    "placeholderSourcePath": "./learn_basis_state_amplitudes/Placeholder.qs",
+    "solutionPath": "./learn_basis_state_amplitudes/solution.md"
+})
+
+
+@[section]({
+    "id": "configure_dumpmachine_output",
+    "title": "Configure DumpMachine Output"
+})
+
+Sometimes you want to focus on certain properties of the quantum state, such as the relative phases of individual basis states, or on certain parts of the state, such as the basis states with larger amplitudes associated with them. 
+[%config](https://docs.microsoft.com/qsharp/api/iqsharp-magic/config) magic command allows you to tweak the format of `DumpMachine` output (available in Q# Jupyter Notebooks only). 
+Here are some useful options that help you extract the right information:
+
+* `dump.basisStateLabelingConvention` sets the way the basis states are labeled in the output. Here is how the basis state $|10\rangle$ will look like with different settings:
+
+  <table style="background-color: white; border:1px solid; tr  { background-color:transparent; }">
+    <col>
+    <col width=300>
+    <tr>
+        <th style="text-align:center; border:1px solid">dump.basisStateLabelingConvention value</th>
+        <th style="text-align:center; border:1px solid">DumpMachine output (first two columns)</th>
```

Comment:
Images are not supported in the current way to compose katas, could you create a text version of the content in the second column?